test/libyul/yulOptimizerTests/rematerialiser/many_refs_small_cost_loop.yul

```diff
@@ -0,0 +1,28 @@
+{
+	// Cost of rematerializating x is 1
+	let x := 0xff
+	// First reference in loop to be rematerialized
```

Comment:
This comment can be changed to

``` Although x has low cost, it is not considered for rematerialization because it is referenced more than 5 times```